index.bs

```diff
@@ -385,9 +385,7 @@ For each message, the sender must write to the QUIC stream the following:
 1.  A type key representing the type of the message, encoded as a variable-length
     integer (see [[#appendix-a]] for type keys)
 
-2.  The message length encoded as a variable-length integer
-
-3.  The message encoded as CBOR (whose length must match the value in step 2)
+2.  The message encoded as CBOR (whose length must match the value in step 2)
```

Comment:
There's no change to the CDDL for this?

Reply:
Nope

Reply:
"step 2" is self referential?  Maybe just drop the phrase in parenthesis?

Reply:
Whoops.  Removed.